app/assets/javascripts/setup.js

```diff
@@ -17,4 +17,32 @@ $(function() {
       $(this).text('Show');
     }
   });
-});
\ No newline at end of file
+
+  $(document).on('click', '.js-toggle-registry-settings-btn', function() {
```

Comment:
You can remove this and add `.js-toggle-overlay-settings-btn` above separating it with a comma.